src/tabular-api-surface.ts

```diff
@@ -736,12 +759,30 @@ Please use the format 'prezzy' or '${instance.name}/tables/prezzy'.`);
         err = null;
       }
 
+      const mutationErrors = Array.from(mutationErrorsByEntryIndex.values());
       if (mutationErrorsByEntryIndex.size !== 0) {
-        const mutationErrors = Array.from(mutationErrorsByEntryIndex.values());
         callback(new PartialFailureError(mutationErrors, err));
         return;
       }
-
+      if (err && timeoutExceeded) {
```

Comment:
instead of checking timeoutExceeded here, I think we can push err to each entry if the entry doesn't have a status code.

Reply:
With the new changes to the test expecting DEADLINE_EXCEEDED I think this makes sense.